charts/sn-console/values.yaml

```diff
@@ -125,6 +125,14 @@ login:
       config:
         AZURE_RESTRICT_DOMAIN_SUFFIXES_NAME: ""
         AZURE_REDIRECT_URI: ""
+    okta:
+      enabled: false
+      # The secret should contain both `OKTA_CLIENT_ID`, `OKTA_CLIENT_SECRET` and `OKTA_TENANT_ID`
+      clientSecretRef: '[kubernetes secret that stores client secret for Okta SSO]'
```

Comment:
Not sure if this secret needs to be created automatically or manually, in the SNP module it is created automatically